Cargo.toml

```diff
@@ -8,11 +8,14 @@ repository = "https://github.com/grovesNL/glyphon"
 license = "MIT OR Apache-2.0 OR Zlib"
 
 [dependencies]
-wgpu = "0.19"
+wgpu = { version = "0.19", default-features = false }
 etagere = "0.2.10"
 cosmic-text = "0.10"
 lru = "0.12.1"
 
 [dev-dependencies]
 winit = { version = "0.29.10", features = ["rwh_05"] }
 pollster = "0.3.0"
+
+[features]
```

Comment:
I think we could probably leave this out and let the application drive wgpu features

Reply:
The main reason that's there is so that the default behavior for wgpu is taken if a glyphon user doesn't do anything. To build specifically for webgl, you have to disable the webgpu feature, but wgpu enables it by default, and the pre-PR way that glyphon was depending on wgpu automatically enabled wgpu's default features. 

This basically just makes it so that a glyphon user who disables default features on glyphon can *also* disable the default features on wgpu. (at least afaik, lmk if my assumption on how features work is incorrect)

Reply:
Yeah that all sounds good. I was just thinking that people can opt-in to wgpu's default features manually instead of asking glyphon to do that (basically removing the `[features]` section here).

In other words, glyphon could always use `wgpu = { version = "0.19", default-features = false }` (since we don't depend on WebGPU/WebGL specifically), and people could still toggle wgpu default features on or off in their `Cargo.toml`s without specifying glyphon features. Would that make sense?

Reply:
yeah, that makes sense, will change